test_opensearchpy/test_helpers/test_document.py

```diff
@@ -648,3 +648,28 @@ class MySubDocWithNested(MyDoc):
         },
         "title": {"type": "keyword"},
     }
+
+
+def test_save_double(mock_client: Any) -> None:
+    class MyDocumentWithDouble(MyDoc):
+        a_double: Union[float, field.Double] = field.Double()
```

Comment:
why is this only float and not `field.Float`?

Reply:
One is a primitive type and the other one is a field.